temoa/extensions/method_of_morris/morris.py

```
@@ -1,28 +1,28 @@
-
-from joblib import Parallel, delayed
-from numpy import array
-from SALib.analyze import morris
-from SALib.sample.morris import sample
-from SALib.util import compute_groups_matrix, read_param_file
-
 seed = 42
 
 
-def evaluate(param_names, param_values, data: dict, k):
+def evaluate(param_names: dict[int, list[Any]], param_values: Any,
```

Ruff (0.14.10)

Dynamically typed expressions (typing.Any) are disallowed in `param_values`

(ANN401)